Author: David Martinez Marti <deavidsedice@gmail.com>
License: GNU GPL, see /usr/share/common-licenses/GPL
Description: fix a bug calculating window size when the orthogonal projection window is set.
Last-Update: 2008-07-22
--- a/src/View.py
+++ b/src/View.py
@@ -139,7 +139,7 @@
       w = int ( viewport[2] - viewport[0] )
       h = int ( viewport[3] - viewport[1] )
       # aspect ratio correction
-      h *= int((float(w) / float(h)) / (4.0 / 3.0))
+      h *= float((float(w) / float(h)) / (4.0 / 3.0))
       viewport = [0, 0, 1, h / w]
   
     if yIsDown:
